### DISCUSSION TITLE
Question for ADA GPU owners.

### DISCUSSION THREAD

**Discussion by Albaricoque695** [hacksider/Deep-Live-Cam#824] [Q&A]
How many FPS do you get?
Other inputs are also appreciated.

**Comment by GuyoST** [comment]
Hello,
How do you make it run with ADA GPU ?
I can make it run correctly with RTX 30xx, but not with RTX 50xx.
I tried to install cuda 12 and ONNX Runtime, but : no face detection.

**Reply by Albaricoque695** [reply]
I haven't, really, but I was considering to buy one if it would make it run more smoothly.

The fact that you have a 5000 series is interesting too, lmk if it's not much of a struggle if you finally manage this to work

**Reply by hacksider** [reply]
> Hello,
> 
> How do you make it run with ADA GPU ?
> 
> I can make it run correctly with RTX 30xx, but not with RTX 50xx.
> 
> I tried to install cuda 12 and ONNX Runtime, but : no face detection.

Please follow the instruction and just use 11.8

**Comment by hacksider** [comment]
There's not much improvement as it is working real time. 16fps on 5090 BUT I'm working on some new stuff that makes it up to 30fps. The reason I can't release it yet is the tech I tried is proprietary.

**Reply by Albaricoque695** [reply]
What tech is it?

**Comment by FurkanGozukara** [comment]
I made it work on RTX 5090 and it is like 22 FPS currently - not utilizing 100% of GPU more like 40% or lesser

**Reply by hacksider** [reply]
Actually 22fps is really awesome! It really doesn't benefit to increase the gpu usage since it usually doesn't benefit that much. Anything beyond 12gb just increase the fps in less than 10% as of this moment since it's already processing frames after the camera gets it.

Though we're experimenting on interpolation and frame generation so it can utilize the gpu faster.

**Reply by hacksider** [reply]
By the way, I'm one of your followers in reddit (cefurkan right?) 😁

**Reply by FurkanGozukara** [reply]
@hacksider yes i am :d thanks for following 

interpolation is a great idea to improve FPS 

by the way i calculate usage based on power usage not VRAM :D

**Reply by Albaricoque695** [reply]
Any trick to improve the quality of the face?, it's kind of blurry
